results/2021/nullified_polling_stations/nullified_polling_stations.csv

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
@morelmiles There is no need to add a folder for the nullified_polling_stations and rename the file to nullified_polling_stations_2020.csv